kb/compiling-and-building/how-do-i-configure-mbedtls.md

```diff
@@ -22,7 +22,7 @@ The `config.py` script automatically finds the `mbedtls_config.h` file when it r
 
 ## Alternative configuration files
 
-You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename (including the quote or angular brackets) at compile time. For example, using **make**:
+You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename at compile time. For example, using **make**: (Note: The angle brackets `<>` are included.)
```

Comment:
Either angle brackets or double quotes are ok: it's what you'd write after an `#include` statement. The current version is missing the information that double quotes are ok.

Reply:
Yep, fair points. Will figure out a smarter wording for this and resubmit.